transport.go

```diff
@@ -363,13 +367,22 @@ func (t *Transport) maybeSendStatelessReset(p *receivedPacket, connID protocol.C
 	if len(p.data) <= protocol.MinStatelessResetSize {
 		return
 	}
+
+	select {
+	case t.statelessResetQueue <- sendStatelessResetArgs{addr: p.remoteAddr, info: p.info, connID: connID}:
+	default:
+		// it's fine to not send a stateless reset when we're busy
+	}
+}
+
+func (t *Transport) sendStatelessReset(addr net.Addr, info *packetInfo, connID protocol.ConnectionID) {
 	token := t.handlerMap.GetStatelessResetToken(connID)
-	t.logger.Debugf("Sending stateless reset to %s (connection ID: %s). Token: %#x", p.remoteAddr, connID, token)
```

Comment:
You can re-parse the connection ID here. Or, for extra performance, even wrap this in a `if t.logger.Debug()`, then we only do the parsing and logging if we're actually logging.

Reply:
I would still need the connId for call to GetStatelessResetToken.